src/openrct2-cli/Cli.cpp

```diff
@@ -1,4 +1,4 @@
-#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
+#pragma region Copyright (c) 2014-2017 OpenRCT2 Developers
```

Comment:
It would be neater, but I'm not sure how long we should be holding this PR back.